### PR TITLE
Test some missing cases

### PR DESCRIPTION
**Pull request by trulsf**

### CODE REVIEW

File: test/runtests.jl [hunk 1]
```diff
@@ -114,9 +115,11 @@ function test_mutable_arithmetics()
     speed = 10u"m/s"
     @test _MA.@rewrite(speed * x) == UnitJuMP.UnitExpression(10x, u"m/s")
     @test _MA.@rewrite(x * speed) == UnitJuMP.UnitExpression(10x, u"m/s")
+    @test _MA.@rewrite(xu * 10) == UnitJuMP.UnitExpression(10x, u"m/s")
     @test _MA.@rewrite(speed * xu) == UnitJuMP.UnitExpression(10x, u"m^2/s^2")
     @test _MA.@rewrite(xu * speed) == UnitJuMP.UnitExpression(10x, u"m^2/s^2")
     @test _MA.@rewrite(x / speed) == UnitJuMP.UnitExpression(0.1x, u"s/m")
+
```
Comment (odow) [inline test/runtests.jl:122]:
```suggestion
```

File: test/runtests.jl [hunk 2]
```diff
@@ -138,8 +141,11 @@ function test_operators()
     @test 400u"m" - xu == UnitJuMP.UnitExpression(-1000x + 400, u"m")
 
     @test 200u"km" * x == UnitJuMP.UnitExpression(200 * x, u"km")
+    @test x * 200u"km"  == UnitJuMP.UnitExpression(200 * x, u"km")
```
Comment (odow) [inline test/runtests.jl:144]:
```suggestion
    @test x * 200u"km" == UnitJuMP.UnitExpression(200 * x, u"km")
```

File: test/runtests.jl [hunk 3]
```diff
@@ -138,8 +141,11 @@ function test_operators()
...
     @test 200 * xu == UnitJuMP.UnitExpression(200 * x, u"km")
+    @test xu * 200  == UnitJuMP.UnitExpression(200 * x, u"km")
```
Comment (odow) [inline test/runtests.jl:146]:
```suggestion
    @test xu * 200 == UnitJuMP.UnitExpression(200 * x, u"km")
```

**Comment by trulsf** [comment]
Thanks, still not familiar with all CI aspects, but enjoying the help and learning a lot.

**Pull request merged**